vscode-antimony/src/server/stibium/stibium/tree_builder.py

```diff
@@ -67,10 +67,12 @@
     'function_call' : FunctionCall,
     'variable_in' : VariableIn,
     'is_assignment' : IsAssignment,
+    'interaction' : Interaction,
+    'interaction_symbol' : Operator,
 }
 
 OPERATORS = {'EQUAL', 'COLON', 'ARROW', 'SEMICOLON', 'LPAR', 'RPAR', 'STAR', 'PLUS', 'MINUS',
-             'DOLLAR', 'CIRCUMFLEX', 'COMMA', 'SLASH', "AEQ", "DBLQUOTE"}
+             'DOLLAR', 'CIRCUMFLEX', 'COMMA', 'SLASH', "AEQ", "DBLQUOTE", "__ANON_0", "__ANON_1", "__ANON_2"}

```

Comment:
ANON0-2 aren't very useful names.  If these are the three interactions, they should probably be called something like 'activate' (-o) 'repress' (-|) and 'interact' ('-(')